velox/exec/tests/PrestoQueryRunnerTest.cpp

```diff
@@ -95,4 +99,52 @@ TEST_F(PrestoQueryRunnerTest, DISABLED_fuzzer) {
   velox::exec::test::assertEqualResults(
       prestoResults, plan->outputType(), {veloxResults});
 }
+
+TEST_F(PrestoQueryRunnerTest, sortedOrder) {
+  auto queryRunner =
+      std::make_unique<PrestoQueryRunner>("http://127.0.0.1:8080", "hive");
```

Comment:
perhaps, change "http://127.0.0.1:8080" to "http://unused" to make it clear that this test doesn't require running Presto instance.